code/modules/client/preference/preferences_mysql.dm

```diff
@@ -55,7 +55,7 @@
 	ooccolor		= sanitize_hexcolor(ooccolor, initial(ooccolor))
 	UI_style		= sanitize_inlist(UI_style, list("White", "Midnight"), initial(UI_style))
 	default_slot	= sanitize_integer(default_slot, 1, max_save_slots, initial(default_slot))
-	toggles			= sanitize_integer(toggles, 0, 2097151, initial(toggles))
+	toggles			= sanitize_integer(toggles, 0, 4194304, initial(toggles))
```

Comment:
Why did you increase this of you didnt add a toggle?
And if you did it should be `4194303`